frontend/public/locales/en/access-to-governmental-benefits.json

```diff
@@ -0,0 +1,71 @@
+      "provincial-or-territorial-dental-benefit": "Provincial or Territorial dental benefit:",
+      "alberta-child-health-benefits": "Alberta Child Health Benefits",
+      "federal-benefits-dental-benefits": "Federal benefits dental benefits:",
+      "dental-service-program": "Dental  Service Program",
+      "update-your-access-text": "Update your access to governmental benefits.",
+      "no-government-benefits": "You have indicated you do not have access to any government benefits",
+      "back": "Back",
+      "links": {
+        "update-your-access-link": "https://www.canada.ca/en/transparency/terms.html"
+      }
+    },
+    "index": {
+      "page-title": "Update your dental benefits",
+      "intro-text": "If you have access to governmental benefits, keep that information up to your date on your application.",
+      "body-text": "This information will be used to help coordinate payment with Sun Life",
+      "update-your-access-text": "Update your access to governmental benefits."
+    },
+    "edit": {
+      "page-title": "Access to other federal, provincial or territorial dental benefits",
+      "access-to-dental": "Access to dental coverage through a provincial, territorial or federal government social program will not impact your eligibility for the Canadian Dental Care Plan.",
+      "select-one": "Select one",
+      "eligibility-criteria": "If you meet all the eligibility criteria, your coverage will be coordinated between the plans to ensure there are no gaps or duplication in coverage.",
+      "required-label": "Complete all fields unless marked as optional",
+      "federal-benefits": {
+        "title": "Federal benefits",
+        "legend": "Do you have dental benefits through a social program offered by a federal program?",
+        "option-yes": "<strong>Yes</strong>, I have <strong>federal</strong> benefits",
+        "option-no": "<strong>No</strong>, I do not have <strong>federal</strong> benefits",
+        "social-programs": {
+          "legend": "Please select which social program you are covered under. If you have more than one, please select the one you use most."
+        }
+      },
+      "provincial-territorial-benefits": {
+        "title": "Provincial or territorial benefits",
+        "legend": "Do you have dental benefits through a social program offered by your province or territory?",
+        "option-yes": "<strong>Yes</strong>, I have <strong>provincial or territorial</strong> benefits",
+        "option-no": "<strong>No</strong>, I do not have <strong>provincial or territorial</strong> benefits",
+        "social-programs": {
+          "input-legend": "If yes, through which province or territory?",
+          "radio-legend": "Please select which social program you are covered under. If you have more than one, please select the one you use most."
+        }
+      },
+      "button": {
+        "back": "Back",
+        "continue": "Continue",
+        "cancel-btn": "Cancel",
```

Comment:
Can we remove "Continue" and "Cancel"? I don't see them in the Figma.

Reply:
Removed: 3e3028732d9cee2c8c61af92f40a0aa376b10515.